interfaces/mathml-core.idl

```diff
@@ -0,0 +1,28 @@
+// GENERATED CONTENT - DO NOT EDIT
+// Content was automatically extracted by Reffy into reffy-reports
+// (https://github.com/tidoust/reffy-reports)
+// Source: MathML Core (https://mathml-refresh.github.io/mathml-core/)
+
+[Exposed=Window]
+interface MathMLElement : Element { };
+MathMLElement includes GlobalEventHandlers;
+MathMLElement includes DocumentAndElementEventHandlers;
+MathMLElement includes HTMLOrForeignElement;
```

Comment:
I tried adding a test for this, but it's blocked on the `HTMLOrForeignElement` mixin not existing. That's blocked on https://github.com/whatwg/html/issues/4702.